napari/settings/_application.py

```diff
@@ -170,6 +181,16 @@ class ApplicationSettings(EventedModel):
         ),
     )
 
+    # convert cache (and max cache) from bytes to mb for widget
+    dask: DaskSettings = Field(
+        default={
+            'enabled': True,
+            'cache': virtual_memory().total * _DEFAULT_MEM_FRACTION / 1e6,
+        },
+        title=trans._("Enable Dask"),
+        description=trans._("Enable/disable Dask caching."),
+    )
```

Comment:
Why did we need to have this as one field, not two?

Reply:
One field will mean one widget.  If it is 2, it will be 2 lines on the preferences dialog.